docker-compose/docker-compose.yaml

```diff
@@ -8,6 +8,10 @@ services:
     networks:
       - backend
       - redis
+    environment:
+      - UVICORN_BASE_URL=${UVICORN_BASE_URL}
+      - USER_API_KEY=${USER_API_KEY}
+      - API_PORT=8000
```

Comment:
with this i think we wont be able to override the API_PORT value.
since we're giving it a default value (https://github.com/ekinolik/stocknear-backend/pull/4/commits/a1acece0b7ca8b48e8cf94a7e0b5a1f438bbd645#diff-990f7e4140fab1ad82afc787505090b64d4024e63a891405cd9085e7e6b249ddR4312), we can pass it just like the others above